x-pack/plugins/security_solution/public/app/home/index.tsx

```diff
@@ -88,10 +96,10 @@ export const HomePage: React.FC<HomePageProps> = ({ children }) => {
 
           {children}
         </DragDropContextWrapper>
-      </Main>
+      </EuiFlexItem>
 
       <HelpMenu />
-    </WrappedByAutoSizer>
+    </EuiFlexGroup>
```

Comment:
So this changes the file shared between all the apps (it's a bit confusing that it's named HomePage). I did a very shallow test but would need to be tested a bit more thoroughly. I guess I should align this change with other teams (maybe Rob?).